geoportal/c2cgeoportal_geoportal/views/dynamic.py

```diff
@@ -90,7 +90,10 @@ def dynamic(self):
             params.update(config.get('params', {}))
             for name, dyn in config.get('dynamic_params', {}).items():
                 params[name] = dynamic[dyn]
-            constants[constant] = self.request.route_url(config['name'], _query=params)
+            constants[constant] = self.request.route_url(config['name'],
+                                                         *config.get('segments', []),
```

Comment:
Why named it segments?
In the pyramid documentations it's named elements ...
https://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.route_url